nuxeo-common/src/main/java/org/nuxeo/common/utils/IdUtils.java

```diff
@@ -35,8 +36,7 @@
 
     public static final String UUID_TYPE_4_REGEXP = "[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}";
 
-    // TODO AT: dummy random, does not ensure uniqueness
-    private static final Random RANDOM = new Random(new Date().getTime());
+    private static final Random RANDOM = new SecureRandom();
```

Comment:
What rule does that refer to?

Reply:
This is squid:S2245 "Using pseudorandom number generators (PRNGs) is security-sensitive"